pallets/relayer/src/lib.rs

```diff
@@ -387,31 +387,44 @@ impl<T: Config> Module<T> {
 
         Ok(())
     }
-}
 
-impl<T: Config> SubsidiserTrait<T::AccountId, T::Balance> for Module<T> {
     fn get_subsidy(
         key: &T::AccountId,
         fee: T::Balance,
-    ) -> Result<Option<T::AccountId>, InvalidTransaction> {
-        // Check if the current paying key matches
+    ) -> Result<Option<Subsidy<T::AccountId, T::Balance>>, InvalidTransaction> {
+        // Get the Subsidy for `key`.
         match <Subsidies<T>>::get(key) {
             // There was no subsidy.
             None => Ok(None),
             // Has subsidy, but not enough remaining POLYX.
             // TODO: Should we fail here or allow falling back to the user key paying?
```

Comment:
I don't think it would be a good idea to silently fall back to the user key paying.  If the user gets an error about payment, then they need to contact their subsidiser.

Reply:
> I don't think it would be a good idea to silently fall back to the user key paying.

Sorry wasn't being clear xD. This is what I think as well.